examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldClientTls.java

```diff
@@ -31,25 +31,42 @@
 import java.util.logging.Logger;
 import javax.net.ssl.SSLException;
 
+// TODO: change flags to use com.google.common.flags
+/*
+import com.google.common.flags.Flag;
+import com.google.common.flags.FlagSpec;
+import com.google.common.flags.Flags;
+*/
+
+
+
 /**
  * A simple client that requests a greeting from the {@link HelloWorldServerTls} with TLS.
  */
 public class HelloWorldClientTls {
     private static final Logger logger = Logger.getLogger(HelloWorldClientTls.class.getName());
 
     private final ManagedChannel channel;
-    private final GreeterGrpc.GreeterBlockingStub blockingStub;
+    private GreeterGrpc.GreeterBlockingStub blockingStub;
+
+    /*
+    @FlagSpec(name = "server_address", help = "address of the server")
+    private static final Flag<String> serverAddr = "localhost:50051";
+    @FlagSpec(name = "server_root_cert_pem_path", help = "path to the root X509 certificate")
+    private static final Flag<String> rootCert = "testdata/ca.cert";
+    @FlagSpec(name = "client_cert_pem_path", help = "path to client's X509 certificate")
+    private static final Flag<String> certFile = "testdata/client.server";
+    @FlagSpec(name = "client_key_pem_path", help = "path to client's private key")
+    private static final Flag<String> keyFile = "testdata/client.key";
+    */
 
-    private static SslContext buildSslContext(String trustCertCollectionFilePath,
-                                              String clientCertChainFilePath,
-                                              String clientPrivateKeyFilePath) throws SSLException {
+    private static SslContext buildSslContext(String clientCertChainFilePath,
+                                              String clientPrivateKeyFilePath,
+                                              String trustCertCollectionFilePath) throws SSLException {
         SslContextBuilder builder = GrpcSslContexts.forClient();
```

Comment:
Can you make this whole method into one line? I.e. `return GrpcSslContexts.forClient().....build()`?